src/lib/components/DeckGL.react.js

```diff
@@ -142,6 +152,7 @@ DeckGL.defaultProps = {
     mapboxKey: null,
     tooltip: false,
     enableEvents: false,
+    disableContext:true,
```

Comment:
I'd suggest false by default or else it'd create confusion. 
```suggestion
    disableContext: false,
```

Reply:
@xhlulu would it cause confusion? I think it's pretty common for elements that use the right mouse button for their own interactivity to disable the context menu. Certainly plotly.js does this (eg https://rreusser.github.io/plotly-mock-viewer/#gl3d_bunny-hull) and at the current low version number I wouldn't worry that the existing behavior is too locked in ;)

Reply:
@alexcjohnson You are right, seems like right click is equivalent to Click+ctrl. I guess it doesn't hurt to set it to default and see how it works out.

Reply:
wow, I didn't even think about using click+crl for orbiting....I see why you were a bit confused by this request at first.